cfgov/v1/util/filterable_list.py

```diff
@@ -116,3 +115,9 @@ def form_id(self):
             return form_ids[0]
         else:
             return 0
+
+    def serve(self, request):
+        """ Modify response header to set a shorter TTL in Akamai """
+        response = super(FilterableListMixin, self).serve(request)
+        response['edge-control'] = 'cache-maxage=10m'
```

Comment:
According to the Akamai docs (behind login wall) they use capitalization `Edge-Control`. They also say:

>  the edge server will issue an If Modified Since request back to the content provider to check whether the object
content has changed

By itself, this might require that we send the `Last-Modified` header with our pages, which I don't believe we do right now (cc @rosskarchner).